jenkins/osa_onmetal.functions.groovy

```diff
@@ -281,6 +281,7 @@ def install_during_upgrade_tests() {
     // Setup during tests
     sh """
     ssh -o StrictHostKeyChecking=no  root@${host_ip} '''
+    mkdir output
```

Comment:
I have a couple of comments related to this. First I would like to use an absolute path, I think it would make it clearer. Secondly, I would prefer this to create the directory only if it does not exist already, that way we can add the same directory creation to the api uptime tests so it does not depend on this one to be installed first to work properly, so replacing that line with the would do:
`mkdir -p /root/output`